### PR TITLE
Prod - Implement multi-language in giddh

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://app.clickup.com/t/1h82ed

### CODE REVIEW

File: apps/web-giddh/src/assets/fonts/icomoon/icomoon.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
     -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-upload-icon:before {
+.icon-upload:before {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/fonts/icomoon/icomoon.css:31]: **icon-upload-icon** do update

Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/fonts/icomoon/icomoon.css:31]: Ignore it is working fine , I checked

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 2]
```diff
@@ -1,168 +1,161 @@
...
+        <div class="row">
+            <div class="col-md-12">
+                <span class="text-light" *ngIf="isUpdateMode">
+                    {{localeData?.adjusted_amount_error}}
+                </span>
+                <div class="row mb-2">
+                    <div class="col-md-8 mr-t05">
                         <div class="form-group">
-                            <label>Amount<span class="text-danger">* </span></label>
+                            <label>{{localeData?.customer_name}} </label>
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html:18]: If you check line numbers, both are at different line numbers.

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 3]
```diff
@@ -1,1561 +1,1433 @@
...
 
-                            <h1 class="mr-t05 font-16 grey-text-color d-flex flex-column">Credit
-                                <span class="digit-blue mt-1">
-                                    {{ ledgerTxnBalance?.currencySymbol}}{{ledgerTxnBalance?.creditTotal | giddhCurrency}}
-                                </span>
-                                <span class="digit-blue font-12 sub-total" *ngIf="isLedgerAccountAllowsMultiCurrency">
-                                    {{ ledgerTxnBalance?.convertedCurrencySymbol}}{{ledgerTxnBalance?.convertedCreditTotal | giddhCurrency}}
-                                </span>
-                            </h1>
+                                    <h1 class="mr-t05 font16 greyTextClr d-flex flex-column">{{commonLocaleData?.app_credit}}
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: Done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**